binder/apt.txt

```diff
@@ -3,3 +3,4 @@ emacs
 ffmpeg
 graphviz
 vim
+texlive-xelatex
```

Comment:
The CI build is failing. According to https://tex.stackexchange.com/a/179811, it should be `texlive-xetex`:
```suggestion
texlive-xetex
```

Reply:
Good catch.